.github/workflows/quality-checks.yml

```diff
@@ -5,8 +5,6 @@ name: quality-checks
 on:
   pull_request:
   push:
-    branches:
-      - main
```

Comment:
@JKatzwinkel, what's the motivation or purpose here?

Reply:
just to get ci feedback without opening a pull request, this can be reverted now that there is one open

Reply:
perfect, thanks!